webcompat/static/js/lib/bugform.js

```diff
@@ -182,18 +193,28 @@ function BugForm() {
     img.src = dataURI;
   };
 
+  // Is the user trying to report a site against webcompat.com itself?
+  this.isSelfReport = function() {
```

Comment:
Passing in the URL to check as a parameter could improve the testability of this function.